src/test/emulators/auth/customToken.spec.ts

```diff
@@ -116,6 +117,44 @@ describeAuthEmulator("sign-in with custom token", ({ authApi }) => {
       });
   });
 
+  it("should not issue a refresh token in passthrough mode", async () => {
+    const uid = "someuid";
+    const claims = { abc: "def", ultimate: { answer: 42 } };
+    const token = signJwt({ uid, claims }, "", {
+      algorithm: "none",
+      expiresIn: 3600,
+
+      subject: "fake-service-account@example.com",
+      issuer: "fake-service-account@example.com",
+      audience: CUSTOM_TOKEN_AUDIENCE,
+    });
+    await updateProjectConfig(authApi(), { usageMode: "PASSTHROUGH" });
+
+    await authApi()
+      .post("/identitytoolkit.googleapis.com/v1/accounts:signInWithCustomToken")
+      .query({ key: "fake-api-key" })
+      .send({ token })
+      .then((res) => {
+        expectStatusCode(200, res);
+        expect(res.body.isNewUser).to.equal(true);
```

Comment:
Updated!

---

Wasn't sure how to respond inline to your other PR comment so also adding it here:

> Can a user provide no string at all? I think if they don't provide any string for usage mode, it makes sense to have it map to default, and if they provide an incorrect string, throw an error. 

Yep! The user can provide no string and it would default to UsageMode.DEFAULT. Added a couple more tests to illustrate the behavior and an additional failing case